docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -44,7 +44,7 @@ so it must complete before the call to delete the container can be sent.
 No parameters are passed to the handler. 
 
 A more detailed description of the termination behavior can be found in
-[Termination of Pods](/docs/user-guide/pods/#termination-of-pods).
+[Termination of Pods](https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods).
```

Comment:
Use the a relative path instead of the full URL so that it will still work in the preview build. Please make the URL `/docs/concepts/workloads/pods/pod/#termination-of-pods` and drop the `https://kubernetes.io` part of it. Thx